libs/sdk-ui-dashboard/src/model/commandHandlers/layout/addSectionItemsHandler.ts

```diff
@@ -37,43 +46,84 @@ function validateAndResolveItems(commandCtx: AddSectionItemsContext) {
         stash,
         items,
         cmd: {
-            payload: { sectionIndex, itemIndex },
+            payload: { itemPath, itemIndex, sectionIndex },
         },
     } = commandCtx;
-    if (!validateSectionExists(layout, sectionIndex)) {
-        throw invalidArgumentsProvided(
-            ctx,
-            commandCtx.cmd,
-            `Attempting to add items to non-existing layout section at index ${sectionIndex}.`,
-        );
-    }
+    if (itemPath === undefined) {
```

Comment:
These handlers handles updated backward compatible commands that has legacy 2D address of item or section (itemIndex, sectionIndex) and new optional itemPath (nested address of item). The handlers do not use feature flag but instead check for new prop(s) on command. If it is undefined, old behavior is used, otherwise new one processes the command. The app state is rewritten to support layout nesting. Even with old behavior the state is updated as if it is nested (just one level deep) and extended events are emitted (both with 2D and nested information).